lib/hal/audio.h

```diff
@@ -34,9 +34,13 @@ typedef struct
 // note that I currently ignore sampleSizeInBits and numChannels.  They
 // are provided to cope with future enhancements. Currently supported samples
 // are 16 bit, 2 channels (stereo)
+__attribute__((deprecated("deprecated as of 05/2021", "Use SDL2 for audio output")))
```

Comment:
*(This is a generic comment about the PR description, that I'm leaving here, so it can be resolved / hidden more easily)*

> Confusingly, JFRs analysis included functions that have already been removed in the past. I took no further actions regarding them.

Such as?

I believe I started looking at master and only used GitHub search rather passively, so I'm not sure while I'd have looked at a wrong revision.